ballerina/Dependencies.toml

```diff
@@ -93,7 +93,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.6.0"
+version = "1.6.1"
```

Comment:
Shouldnt this be a minor release version rather than a patch version? @niveathika @aashikam

Reply:
Yes yes

Reply:
This is io package dependency!

Reply:
https://github.com/ballerina-platform/module-ballerinax-salesforce/blob/ca9a5787ce72700e78db7cfa43efc742ed164f2b/gradle.properties#L3 

Salesforce new version: 8.1.0 
Last version in central: 8.0.2